### PR TITLE
shapeshifter recipe added

### PR DESCRIPTION
**Pull request by frytime32**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/shapeshifter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dea83a20033b75e35cd4483b8703d322f81315a448133615f9254d75a03c1924
+
+build:
+  skip: True # [py27]
```
Comment (bgruening) [inline recipes/shapeshifter/meta.yaml]: Can you please make this a `noarch: python` package?

File: recipes/shapeshifter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pandas
```
Comment (bgruening) [inline recipes/shapeshifter/meta.yaml]: - pip
- python >=3

and the rest can be dropped from the host requirements imho.

File: recipes/shapeshifter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - shapeshifter.files
+    - shapeshifter.utils
+
+about:
+  home: https://github.com/srp33/ShapeShifter
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: A tool for managing large datasets
+  doc_url:
```
Comment (bgruening) [inline recipes/shapeshifter/meta.yaml]: can you remove those as well

File: recipes/shapeshifter/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - shapeshifter
+    - shapeshifter.errors
+    - shapeshifter.files
+    - shapeshifter.utils
+
+about:
+  home: https://github.com/srp33/ShapeShifter
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (bgruening) [inline recipes/shapeshifter/meta.yaml]: LICENSE

**Comment by frytime32** [comment]
@bgruening As far as I can tell, everything is ready for merging! Let me know if anything else needs to be changed

**Comment by frytime32** [comment]
@bgruening I don't have write access, do I need to do anything to get this merged? Or do I just wait now? I just want to make sure I'm not missing something.

**Comment by stain** [comment]
Thanks, @frytime32! Merged now.

**Pull request merged**